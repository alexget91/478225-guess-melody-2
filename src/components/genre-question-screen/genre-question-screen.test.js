import React from "react";
import renderer from "react-test-renderer";
import GenreQuestionScreen from "./genre-question-screen";
import createAudioMock from "../../common/test-stubs";

it(`Genre question screen correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<GenreQuestionScreen
      screenIndex={0}
      question={{
        type: `genre`,
        genre: ``,
        answers: [{
          genre: ``,
          src: ``,
        }],
      }}
      onAnswer={jest.fn()}
    />, createAudioMock())
    .toJSON();

  expect(tree).toMatchSnapshot();
});
