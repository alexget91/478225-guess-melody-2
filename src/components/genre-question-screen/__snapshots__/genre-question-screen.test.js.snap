// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genre question screen correctly renders after relaunch 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Выберите 
    
     треки
  </h2>
  <form
    className="game__tracks js-answer-form"
    onSubmit={[Function]}
  >
    <div
      className="track"
    >
      <button
        className="track__button track__button--play js-track-button"
        disabled={true}
        onClick={[Function]}
        type="button"
      />
      <div
        className="track__status"
      >
        <audio />
      </div>
      <div
        className="game__answer"
      >
        <input
          className="game__input visually-hidden"
          id="answer-0"
          name="answer"
          onChange={[Function]}
          type="checkbox"
          value="answer-0"
        />
        <label
          className="game__check"
          htmlFor="answer-0"
        >
          Отметить
        </label>
      </div>
    </div>
    <button
      className="game__submit button"
      type="submit"
    >
      Ответить
    </button>
  </form>
</section>
`;
